src/app/components/FacetSidebar/FacetSidebar.jsx

```diff
@@ -151,7 +151,7 @@ class FacetSidebar extends React.Component {
 
         if (facet.field === 'date') {
           return (
-            <div className={`nypl-facet-search nypl-spinner-field ${this.state.spinning ? 'spinning' : ''}`}>
+            <div key={`${facet.field}-${facet.value}`} className={`nypl-facet-search nypl-spinner-field ${this.state.spinning ? 'spinning' : ''}`}>
```

Comment:
Oh yeah, it seemed like that was declared but not really used! I updated it here and in the rest of the loop.